packages/errors/src/react/error-page.tsx

```diff
@@ -98,14 +97,16 @@ const ErrorPage: React.FC<ErrorPageProps> = props => {
         <div {...styles.icon(themeName)}>{icons[iconId]()}</div>
       )}
       <div {...styles.text()}>
-        <Heading size={headingSizes.medium}>
+        <Heading size={Heading.sizes.medium}>
           <h1>{text}</h1>
         </Heading>
       </div>
       <div {...styles.code(themeName)}>
-        <Heading size={headingSizes.smallCaps}>
-          <h2>Error code: {code}</h2>
-        </Heading>
+        <h2>
```

Comment:
@jaketrent This is a good point. The `<Heading>` component has been indepependent of the `<h1>, <h2>, <h3>, ...` it's used with. Perhaps the `<Heading>` should have numeric levels that correspond to the `<h1>, <h2>, <h3>, ...` tags and then the `<Label>` component could comprise all of the sizes of the typographic scale and be utilized inside the `<Heading>` and `<P>`components.

Reply:
If `Label` includes all typographic scales/styles, then I think we can do without probably all other Components because a user could just bring their own semantics.  And, in this scenario, maybe it's called something more generic:

```
<Text size="thatExtraBigOneForHeadings">
  <h1>Title</h1>
</Text>

<Text size="thatMidSizeForParas">
  <p>text...</p>
</Text>

<Text size="thatMidSizeForParas">
  <span>label</span>
</Text>

<Text size="thatSmallishForEyebrows" caps>
  <div>Raise your</div>
</Text>

<Text size="thatMidSizeForParas" mono>
  <code>Raise your</code>
</Text>
```

And an alternate syntax which could produce the same thing:

```
<Text.h1 size="thatExtraBigOneForHeadings">
  Title
</Text.h1>

<Text.p size="thatMidSizeForParas">
  text...
</Text.p>

<Text.span size="thatMidSizeForParas">
  label
</Text.span>

<Text.div size="thatSmallishForEyebrows" caps>
  Raise your
</Text.div>

<Text.code size="thatMidSizeForParas">
  Raise your
</Text.code>
```

^ This is a very coder-centric view of the debate. Semantics aren't readily apparent in most product designs.

Regarding the other specific idea of creating 1-6 levels on `Heading` to match `h1-6`.  I believe that this would confuse and combine what we have tried to keep separate historically:

- Heading sizes are in English and only denote visual size
- Markup sizes are 1-6 and only denote semantics

And you will need to mix/match according to the context of your app, which we don't know enough about to anticipate.